### ISSUE TITLE
Implement GetCurrentRTMPSettings

### ISSUE THREAD

**Issue by lindea** [haganbmj/obs-websocket-js#48]
### Is this a request for a new *feature* or a *bug* report?
feature

### Description:
The obs-websocket has implemented GetCurrentRTMPSettings. 
Would be great if it was added here as well.

https://github.com/Palakis/obs-websocket/blob/rtmp-settings/PROTOCOL.md#getcurrentrtmpsettings

**Comment by haganbmj** [comment]
Yep. You can send that request with `.send('GetCurrentRTMPSettings')`  

I'll add it to the list of bound commands later today so that you can do `.getCurrentRTMPSettings` if that's your preference.

**Comment by haganbmj** [comment]
Added it so you can access it via any of the following. I'd recommend the first or second. The third probably shouldn't even be an option.

```
.send('GetCurrentRTMPSettings')
.getCurrentRTMPSettings()
.GetCurrentRTMPSettings()
```

**Comment by haganbmj** [comment]
Available through bower and node @v0.5.3

**Comment by lindea** [comment]
Great! Thank you so much.
